### ISSUE TITLE
u2212 UnicodeEncodeError when using soup.prettify()

### ISSUE THREAD

**Issue by leeuwe** [adiralashiva8/robotframework-metrics#94]
Trace
```>robotmetrics
INFO:Converting .xml to .html file. This may take few minutes...
INFO:1 of 4: Capturing dashboard content...
INFO:2 of 4: Capturing suite metrics...
INFO:3 of 4: Capturing test metrics...
INFO:4 of 4: Capturing keyword metrics...
Traceback (most recent call last):
  File "C:\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Python39\Scripts\robotmetrics.exe\__main__.py", line 7, in <module>
  File "C:\python39\lib\site-packages\robotframework_metrics\runner.py", line 115, in main
    generate_report(args)
  File "C:\python39\lib\site-packages\robotframework_metrics\robotmetrics.py", line 955, in generate_report
    outfile.write(soup.prettify())
  File "C:\python39\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u2212' in position 74783: character maps to <undefined>

Installed packages:

Package                                Version
-------------------------------------- ----------
beautifulsoup4                         4.10.0
robotframework                         4.1
robotframework-metrics                 3.2.2
soupsieve                              2.2.1

**Comment by adiralashiva8** [comment]
I guess this will fix issue - https://github.com/adiralashiva8/robotframework-metrics/issues/92#issuecomment-921771115

**Comment by leeuwe** [comment]
Confirmed, this does fix the problem

**Comment by leeuwe** [comment]
Will be closed by #92
